lib/src/vector_math/matrix4.dart

```diff
@@ -638,20 +638,29 @@ class Matrix4 {
   Matrix4 operator -() => clone()..negate();
 
   /// Returns a new vector or matrix by multiplying this with [arg].
+  ///
+  /// [arg] should be a [double] (to scale), [Vector4] (to transform), [Vector3]
+  /// (to transform), or [Matrix4] (to multiply).
+  ///
+  /// If you know the argument type in a call site, prefer [scaled],
+  /// [transformed], [transformed3], or [multiplied] for performance.
+  @pragma('wasm:prefer-inline')
+  @pragma('vm:prefer-inline')
+  @pragma('dart2js:prefer-inline')
   dynamic operator *(dynamic arg) {
+    final dynamic value;
```

Comment:
Two nits:
1. I prefer Martin's suggestion of `result` - the reader already knows what the variable is for.
2. Prefer `Object` over `dynamic` for the variable - it signals that you don't intend to use the result via a dynamic call.

Reply:
Done.